inherits.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function inherits(ctor, superCtor) {
```

Comment:
does it used anywhere now ? in first commits it was referenced from vite config, now I can't find any mentions

Reply:
yeah can be removed